confighelper/beaconbroadcaster.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2017-2018 Joel Allen Luellwitz and Andrew Klapp
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+""" Provides the broadcasting component of a a filesystem-based IPC mechanism."""
+
+__all__ = ['BeaconBroadcasterBroadcastException',
+           'BeaconBroadcasterInitException',
+           'BeaconBroadcaster']
+
+import datetime
+import logging
+import os
+import stat
+import tmpfs
+
+GROUP_RW_MODE = stat.S_IXUSR | stat.S_IRUSR | stat.S_IWUSR | stat.S_IXGRP | stat.S_IRGRP
```

Comment:
I don't like defining constants at the top of the file just because. Generally I like to declare stuff close to where it is used.

I try to only define constants at the top of a file if I feel they are likely to change (although I admit I might not be following that very carefully for Parkbench stuff).